features/resources/v2algodclient_responsejsons/accountInformationMinBalance.json

```diff
@@ -0,0 +1,13 @@
+{
+  "pending-rewards" : 0,
+  "created-assets" : [],
+  "assets" : [],
+  "address" : "ALGORANDAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAIN5DNAU",
+  "amount-without-pending-rewards" : 1500000,
+  "amount" : 1500000,
+  "status" : "Not Participating",
+  "round" : 6222133,
+  "reward-base" : 0,
+  "rewards" : 0,
+  "min-balance" : 100000
```

Comment:
```suggestion
  "min-balance" : 1377000
```

Reply:
I'm proposing a surprising (non-sensical) min-balance amount for this fixture since the fixture should be parsed as is with no deep validations beyond the field type taking place.